src/main/java/edu/wisc/my/ltiproxy/web/LTIController.java

```diff
@@ -30,13 +35,35 @@ public void setLTILaunchService(LTILaunchService LTILaunchService){
       this.LTILaunchService = LTILaunchService;
   }
 
+  @RequestMapping(value="/go/{key}", method=RequestMethod.GET)
+  public void proxyRedirect(HttpServletRequest request, HttpServletResponse response, @PathVariable String key) throws 
+          ClientProtocolException, IOException, LtiSigningException, URISyntaxException {
+      int statusCode = HttpServletResponse.SC_BAD_REQUEST;
```

Comment:
Suggest inline `HttpServletResponse.SC_BAD_REQUEST` the one place it's used in this method.